### PR TITLE
Update PNI share copy + image

### LINKED ISSUE #3785: Update PNI OG copy and image
This is an dev ticket to update the OG graphic and copy. Currently blocked by:

@natalieworth to add image here

@caltrider to add copy here

### PR DESCRIPTION
**Pull request by youriwims**
Closes #3785

**Dev + Design Review**
---
Although you will see updated copy and image if you share the app link on facebook/twitter/email right now, you **wont** see it updated if you check the pni product vote share buttons because the url is hitting our prod url. 

Once this lands and is pushed to prod, I'll double check that both work properly on prod.

### CODE REVIEW

**Comment by youriwims** [comment]
Resource: https://developers.facebook.com/docs/sharing/webmasters/images/#precaching
Share Preview once on prod:
Screen Shot 2019-11-06 at 2 24 43 PM

**Comment by kristinashu** [comment]
URL shares looks good in slack too 

image


### Just to clarify...

> Although you will see updated copy and image if you share the app link on facebook/twitter/email right now, you wont see it updated if you check the pni product vote share buttons because the url is hitting our prod url.

Right now it looks like this:
image
We want to keep "I think this [product name] is [rating] CREEPY. What do you think? Check out the Creep-O-Meter over on @mozilla’s PrivacyNotIncluded.org buyer’s guide." the same. Just the pink image and OG text will be new, right?

**Comment by youriwims** [comment]
>We want to keep "I think this [product name] is [rating] CREEPY. What do you think? Check out the Creep-O-Meter over on @mozilla’s PrivacyNotIncluded.org buyer’s guide." the same. Just the pink image and OG text will be new, right?

Correct 👍

**Comment by kristinashu** [comment]
Ok cool cool.

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 1]
```diff
@@ -10,11 +10,11 @@
     {% endblock %}
     <meta property="og:type" content="website">
     {% block head_extra %}
-    <meta property="og:title" content="*Privacy Not Included: A Buyer’s Guide for Connected Products">
+    <meta property="og:title" content="*Privacy Not Included: A Buyer's Guide for Connected Products">
     {% endblock %}
     <meta name="csrf-token" content="{{ csrf_token }}">
-    <meta property="og:description" content="89 connected products rated on their safety and security">
-    <meta property="og:image" content="{{request.scheme}}://{{request.get_host}}/_images/buyers-guide/share.jpg">
+    <meta property="og:description" content="Smart home gadgets, fitness trackers, toys and more, rated for their privacy & security">
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: Oh good, we got away from the numbers

**Pull request merged**